### PR TITLE
Draft: Move model2nnpkg install process

### PR DESCRIPTION
**Pull request by hseok-oh**
- Install in one-cmd's cmake
- Remove install in rules & packaging preset

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: compiler/one-cmds/CMakeLists.txt [hunk 1]
```diff
@@ -74,6 +74,15 @@ foreach(ONE_UTILITY IN ITEMS ${ONE_UTILITY_FILES})
...
 endforeach(ONE_UTILITY)
 
+# one-pack internally uses model2nnpkg tool
+set(MODEL2NNPKG "${NNAS_PROJECT_SOURCE_DIR}/tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh")
+install(FILES ${MODEL2NNPKG}
+              PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE
+                          GROUP_READ GROUP_EXECUTE
+                          WORLD_READ WORLD_EXECUTE
+              DESTINATION bin
+              RENAME "model2nnpkg")
```
Comment (hseok-oh) [inline compiler/one-cmds/CMakeLists.txt:84]: If we don't change name, we don't need to modify `compiler/one-cmds/one-pack` and `infra/debian/compiler/one-compiler.install`, and `infra/packaging/res/tf2nnpkg.20221125` files.

**Comment by hseok-oh** [comment]
@mhs4670go @seanshpark Is it acceptable change?

**Comment by mhs4670go** [comment]
Looks good to me.

**Comment by hseok-oh** [comment]
OK. I'll make PRs for this change.

**Comment by hseok-oh** [comment]
Done

**Comment by seanshpark** [comment]
@hseok-oh , `model2nnpkg.sh` is used in `one-cmds`
```
# one-pack internally uses model2nnpkg tool
set(MODEL2NNPKG "${NNAS_PROJECT_SOURCE_DIR}/tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh")
```

is this OK?

**Comment by hseok-oh** [comment]
I forgot. Thankss.

**Pull request closed without merging**